common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -563,14 +563,16 @@ def definition_to_xml(self, resource_fs):
                 if key in self.fields and self.fields[key].is_set_on(self):
                     try:
                         xml.set(key, unicode(value))
-                    except ValueError as exception:
-                        exception_message = "{message}, Block-location:{location}, Key:{key}, Value:{value}".format(
-                            message=exception.message,
-                            location=unicode(self.location),
-                            key=key,
-                            value=unicode(value)
-                        )
-                        raise ValueError(exception_message)
+                    except UnicodeDecodeError:
+                        exception_message = exception_message_for_xml(self.location, key, value)
+                        log.exception(exception_message)
+                        # If exception is UnicodeDecodeError set value using unicode 'utf-8' scheme.
+                        log.info("Setting xml value using 'utf-8' scheme.")
```

Comment:
log.exception means we got error and log.info for notify that we are making other attempt using utf-8 scheme.